index.html

```diff
@@ -4580,6 +4580,340 @@ <h1>Current Issues</h1>
 
   </section>
 
+  <section class="appendix">
+      <h1>Frequently Asked Questions about DID Identification</h1>
```

Comment:
This PR is looking more and more like something that shouldn't go in the base specification, but rather an implementation guide or primer... Specs typically don't have FAQs in them. We can keep this here for now since we have nowhere else to put it.

Reply:
@msporny I was thinking the same thing and actually played with different names for this section. Happy to change the main heading and reword the subheads so they are not questions.